src/index.ts

```diff
@@ -20,12 +23,32 @@ app.get("/online-archaeologists", (req: Request, res: Response) => {
     .catch(() => res.status(500));
 });
 
-app.get("/uptime-stats", (req: Request, res: Response) => {
+app.get("/arch-uptime-statistics", (req: Request, res: Response) => {
   getUptimeStats()
     .then(stats => res.send(stats))
     .catch(() => res.status(500));
 });
 
+app.get("/bundlr/publicKey", async (req: Request, res: Response) => {
+  const key = process.env.BUNDLR_PAYMENT_PRIVATE_KEY!;
+  if (!key) throw new Error("Private key is undefined!");
+
+  const signer = new TypedEthereumSigner(key);
+  res.status(200).json({ publicKey: signer.publicKey.toString("hex") });
+});
+
+app.post("/bundlr/signData", async (req: Request, res: Response) => {
+  const messageDataBuffer = Buffer.from(req.body.messageData, "hex");
+
+  const key = process.env.BUNDLR_PAYMENT_PRIVATE_KEY;
+  if (!key) throw new Error("Private key is undefined!");
+
+  const signer = new TypedEthereumSigner(key);
+  const signature = Buffer.from(await signer.sign(messageDataBuffer));
+
+  res.status(200).json({ signature: signature.toString("hex") });
+});
```

Comment:
We will need a way to gate this to specific domains (starting `app.dev.sarcophagus.io` and `app.sarcophagus.io`), possibly with the request header, and we may also need to determine if there's any other further validation on the message data needed (possibly not if Bundlrs examples dont show anything else).